ion-schema/src/isl/mod.rs

```diff
@@ -85,26 +86,43 @@ pub mod util;
 /// Provides an internal representation of an schema file
 #[derive(Debug, Clone)]
 pub struct IslSchema {
-    // Represents all the IslImports inside the schema file.
-    // For more information: https://amazon-ion.github.io/ion-schema/docs/isl-1-0/spec#imports
+    /// Represents all the IslImports inside the schema file.
+    /// For more information: https://amazon-ion.github.io/ion-schema/docs/isl-1-0/spec#imports
     imports: Vec<IslImport>,
-    // Represents all the IslType defined in this schema file.
-    // For more information: https://amazon-ion.github.io/ion-schema/docs/isl-1-0/spec#type-definitions
+    /// Represents all the IslType defined in this schema file.
+    /// For more information: https://amazon-ion.github.io/ion-schema/docs/isl-1-0/spec#type-definitions
     types: Vec<IslType>,
-    // Represents all the inline IslImportTypes in this schema file.
+    /// Represents all the inline IslImportTypes in this schema file.
     inline_imported_types: Vec<IslImportType>,
+    /// Represents open content as `Element`s
+    /// Note: This doesn't preserve the information about where the open content is placed in the schema file.
```

Comment:
Can you document _why_ you have chosen to do it this way? It's possible that in a year, there will be a use case that requires the relative ordering of ISL and non-ISL. If that comes up, we need to understand why the decision was made so that we can make an informed decision about whether to change it.

Reply:
As of now I do not know of any use case for preserving this information and it requires storing the order for non ISL and ISL which could complicate the current `IslSchema` model by having to store the order and add a model to represent non ISL data. If a use case arrives we can change it.